geodiff/src/drivers/postgresdriver.cpp

```diff
@@ -798,6 +818,10 @@ void PostgresDriver::applyChangeset( ChangesetReader &reader )
       throw GeoDiffException( "Unexpected operation" );
   }
 
+  // at the end, update any SEQUENCE objects if needed
+  for ( auto it : autoIncrementTablesToFix )
```

Comment:
i do not like auto... is it const or not?

Reply:
changed in b1b2982cb09040fafa689faf0243ded1f3b1a19b to `const std::pair<std::string, int64_t> &`